client/modules/IDE/pages/IDEView.jsx

```diff
@@ -34,6 +34,13 @@ import AddToCollectionList from '../components/AddToCollectionList';
 import Feedback from '../components/Feedback';
 import { CollectionSearchbar } from '../components/Searchbar';
 
+const getTitle = (props) => {
```

Comment:
Small code style point: I prefer to use function syntax for top level functions because I think it's more readable!

Reply:
Sure thing Cassie!